frontend/lib/uast/post-parse-checks.cpp

```diff
@@ -294,7 +294,7 @@ void Visitor::checkNoDuplicateNamedArguments(const FnCall* node) {
       if (!actualNames.insert(name).second) {
         auto actual = node->actual(i);
         assert(actual);
-        error(actual, "The named argument '%s' is used more "
+        error(actual, "the named argument '%s' is used more "
```

Comment:
I see several other errors in this file that are capitalized, as well as some ending in periods and some not -- if changing this is in the scope of this PR, the others should probably all be made consistent as well.

Reply:
I've gone through and updated all the errors to lowercase-with-punctuation format.